cve_bin_tool/parsers/dart.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from cve_bin_tool.parsers import Parser
+
+
+class DartParser(Parser):
+    """
+    A parser class for handling pubspec.lock file
+
```

Comment:
Would be great to have a link to the pubspec.lock format here.

Reply:
And maybe a link to dart too, since I wouldn't assume most people know about it the way I expect them to have heard of perl or python.